### PR TITLE
feat(grafana): sort stages

### PR DESCRIPTION
**Pull request by jeremie-H**
In the grafana dashboard, I think it could be easier to follow the progress if stages are in the right order
## Before
image

## After
image

### CODE REVIEW

File: etc/grafana/dashboards/overview.json [hunk 1]
```diff
@@ -249,6 +249,36 @@
...
+              "Execution": 4,
+              "Finish": 12,
+              "Headers": 0,
+              "IndexAccountHistory": 11,
+              "IndexStorageHistory": 10,
+              "MerkleExecute": 8,
+              "MerkleUnwind": 5,
+              "SenderRecovery": 3,
+              "StorageHashing": 7,
+              "Time": 13,
```
Comment (rkrasiuk) [inline etc/grafana/dashboards/overview.json:275]: why is `Time` needed here?

Comment (jeremie-H) [reply etc/grafana/dashboards/overview.json]: You're right, it's automatically added, but I've removed this field

**Review by onbjerg — APPROVED** [review]
lgtm!!

**Comment by jeremie-H** [comment]
Hello, 
@onbjerg, I suggest to reopen and reapply this MR, my work was delete by @shekhirin on this MR : https://github.com/paradigmxyz/reth/pull/4385

**Pull request merged**